### PR TITLE
make state colors of articles/pages consistent

### PR DESCRIPTION
**Pull request by penk85**
WPC-533: use a single color enum for states across the editor.

### CODE REVIEW

File: packages/editor/src/client/routes/articleEditor.tsx [hunk 1]
```diff
@@ -409,7 +409,7 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     <>
       <fieldset style={{borderColor: stateColor}}>
         <legend style={{width: 'auto', margin: '0px auto'}}>
-          <Tag color={stateColor}>{tagTitle}</Tag>
+          <Tag style={{backgroundColor: stateColor}}>{tagTitle}</Tag>
```
Comment (penk85) [inline packages/editor/src/client/routes/articleEditor.tsx:412]: @bishoymelek I think this resolves the Rsuite color warning

**Review by camilleory — APPROVED** [review]

File: packages/editor/src/client/global.less [hunk 2]
```diff
@@ -7,19 +7,19 @@
   &-row.approved:not(&-row-header),
   &-row.approved &-cell-group,
   &-row.approved &-cell {
-    background: rgb(225, 248, 222);
+    background: #e1f8de;
```
Comment (roosnic1) [inline packages/editor/src/client/global.less:10]: are those entries still needed? And if yet where?

**Review by roosnic1 — COMMENTED** [review]
looks good but i'm not sure about the css in the `global.less` file

**Review by lakeside763 — APPROVED** [review]

File: packages/editor/src/client/global.less (hunk 2 above)
Comment (penk85) [reply packages/editor/src/client/global.less:10]: @roosnic1 this is for the colors on the comment list. I thought it's better if it's the same colors throughout the editor.

Comment (roosnic1) [reply packages/editor/src/client/global.less:10]: 👍🏾

**Review by roosnic1 — APPROVED** [review]
🎉

**Review by bishoymelek — APPROVED** [review]
🎉  good one

**Pull request merged**